README.md

```diff
@@ -1379,10 +1379,10 @@ if err := snapshot.PackFrom("my-snapshot.tgz", scratchDir); err != nil {
 
 ## Calling external programs
 
-By default `ghw` may call external programs, for example `ethtool`, to learn
-about hardware capabilities.  In some rare circumstances it may be useful to
-opt out from this behaviour and rely only on the data provided by
-pseudo-filesystems, like sysfs.
+By default ghw may call external programs, to learn about hardware capabilities.
+The general direction is to gather the information using go packages and/or
+system libraries, but ghw still relies on tools especially on less maintained
+platforms or architectures (e.g. Darwin).
```

Comment:
```suggestion
We gather information using Go packages and system libraries, but in some cases ghw
may call external tools/binaries, particularly on non-Linux platforms.
```

Reply:
done, thanks

Reply:
@ffromani I don't see you have made the above changes? Did you forget to push a commit?

Reply:
@jaypipes yes sorry, I answer on github when I committed local changes on my tree, didn't push yet. Will do shortly.